pkg/volume/glusterfs/glusterfs.go

```diff
@@ -145,6 +145,19 @@ func (plugin *glusterfsPlugin) execCommand(command string, args []string) ([]byt
 	return cmd.CombinedOutput()
 }
 
+func (plugin *glusterfsPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.Spec, error) {
+	glusterfsVolume := &api.Volume{
+		Name: volumeName,
+		VolumeSource: api.VolumeSource{
+			Glusterfs: &api.GlusterfsVolumeSource{
+				EndpointsName: volumeName,
+				Path:          volumeName,
```

Comment:
Ack. Yes, that's fine then. And yes it is, since they get set to default values.